pkg/controller/velero/velero.go

```diff
@@ -582,3 +581,46 @@ func credentialsRequestSpecEqual(x, y minterv1.CredentialsRequestSpec) (bool, er
 
 	return true, nil
 }
+
+// GenerateServiceMonitor generates a prometheus-operator ServiceMonitor object
+// based on the passed Service object.
+func generateServiceMonitor(s *corev1.Service) *monitoringv1.ServiceMonitor {
```

Comment:
if this is a function that we're going to own then we should have a unit test for it imo